model/src/main/kotlin/config/FileArchiverConfiguration.kt

```diff
@@ -37,6 +37,11 @@ import org.ossreviewtoolkit.utils.core.storage.LocalFileStorage
  */
 @JsonIgnoreProperties(value = ["patterns"])
 data class FileArchiverConfiguration(
+    /**
+     * Toggle to enable or disable the file archiver functionality altogether.
+     */
+    val enabled: Boolean = true,
```

Comment:
Please add this also to `reference.conf`.

Reply:
Thanks for reminding me, done.